features/configure-clocksource/configure.d/configure-clocksource

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
```

Comment:
Other feature profiles have their scripts stored in the `scripts/` directory, and then symlink them to the relevant profile hook directories. Please could we be consistent with that?